build-utils/optional-locale-chunk-plugin.js

```diff
@@ -10,18 +10,24 @@
  * This plugin removes the locale chunks from the app entrypoint's immediate
  * chunk dependants list, ensuring that the compiled entrypoint will execute
  * *without* all locale chunks loaded.
+ *
+ * Note that the locale actually gets initially loaded in the server template and
+ * the frontend code (in `app/translations`) may be redundant.
  */
 const PLUGIN_NAME = 'OptionalLocaleChunkPlugin';
 
-const clearLocaleChunks = chunks =>
-  chunks
+const clearLocaleChunks = chunks => {
```

Comment:
Intentionally did this so it's a bit more clear that we're not expecting a return value.